test/algorithms/buffer/buffer_linestring_geo.cpp

```diff
@@ -116,9 +116,11 @@ void test_linestring_aimes()
     std::set<int> const skip_cases_miter_flat{};
 #else
     // Cases where the algorithm is still failing.
-    std::set<int> const skip_cases_round_round{17, 22, 38, 181, 196};
-    std::set<int> const skip_cases_round_flat{17, 22, 38, 103, 196};
-    std::set<int> const skip_cases_miter_flat{17, 18, 22, 38, 103, 196};
+    // It might differ in debug/release mode.
+    // It might also depend on the compiler and on the operating system.
+    std::set<int> const skip_cases_round_round{17, 22, 38, 67, 75, 109, 163, 181, 143, 196};
+    std::set<int> const skip_cases_round_flat{17, 22, 38, 67, 75, 103, 109, 196};
+    std::set<int> const skip_cases_miter_flat{17, 18, 22, 38, 67, 75, 103, 109, 196};
```

Comment:
A few cases were added - not (necessarily) by this PR but by the fact that earlier I tested in release mode, and now in debug mode. I also added `67` and `163 reported by @tinko92